.kitchen.ec2.yml

```diff
@@ -43,67 +43,50 @@ platforms:
       - recipe[curl]
     driver:
       image_id: ami-2a4d1a40
-    transport:
```

Comment:
We have used it to specify the username for the given AMI on EC2. Apparently, now the kitchen-ec2 gem selects the proper username automatically.